.travis.yml

```diff
@@ -17,6 +17,9 @@ env:
         # Add the APPVEYOR_TOKEN environment variable for the "conda-forge" AppVeyor user.
         - secure: "GROiBt5zib8ZYg2CqLJMdfIRgFxqYX7rPbzedtHU7L+YTEsacj/HbbzEwWVysMwDoMs5MWGlOoK6c0yCZEVrD5WisE2PN3xLTNzJkNSqPyHH4SLY3+LgVo6416sP3kNh3xftmeyY5pWSX3WhFLaI+BdPZNSYSatfcQcqsJRn3KHeWaDwUFjYxhJ4jVsZB+ELFQAlLU3gpC5yCKXOaB9BMwLSJai8a9BGpfSFjkOzr9RgeAdaZutHYEmmHpv91sjtS/GC9Jw6991L/58KWHqe+NxURg3HIziYWoZxYiRFojN4A9Bn+n4FbzWOA/ELOG+D95P9JeYpq+IS3g0j2a14J9UQK7sz6Rqa/niO7pkYBPn1Wp+rOyaw1Ietmcnu3OzK6cD8BBQH0D/4IRxNDgBKf3oUyt2fVAqgpm6LQ//yUaD7alvj7Z0oaGC1MQwooiF/3Bz6fcxE+dmIECVsDFrJ2+sBad3gm2fuHP0Ng33ZyoIn0ehu2Byrc0FB6/lXYUmIDxEhpQQDL0Qs6IBQV5uXolPrDZAhOlhzLg4OFvDABkUFehwroHJSRRjtSU8Ok0GAiiaInteMatTvmoyPYJ8TM+fElOqXnZU07UlZ4cqCM1v2MyZE4JOW0gR6lBm7XjbI1jbo2Etf+CQMYCf6f2HcnTBAHhvBIxJWaJRAWYVnq0c="
 
+        # Add the TRAVIS_TOKEN environment variable for the "conda-forge" Travis user.
+        - secure: "BBJrw5OcEG4zEUxjXt2txNoz+o7ilTb0e8Tj1626USKqi6HAWOdSFHW7PLYR6vyefWsHnGLAeYMRr9UB56O2oDwS7jgClBKbyut2+Q+8SQBwEtNCMHdU996IzSuDP4AM3h7DLjTsuinWhwpqLu72z2ZjE9OMz4q03SyyiBOecWU1btYbp/u3bpNzyhb2u0HMkRUnS8MxPUa8aHZBKp/JpLrGQmIT0LcjLNp4/jTZp+cs+J//02hAhJHV3vMmyefVvRZayu3xKzutpb3qIga0JAP3d0j2FWXRHVTW6Ke5H4u/4NU7iTpdRjpymnzH7H2kfOrNzgCJ23NfNhScDJDc5URvdXlstgkC4lDkd7BbSVgKKaVy2vVcMkz98qbBnhi0akMHNZXPLvT9tec9AsAX5/pdNaa1rH/iRhjWRfTCnqODX+RTwMhA1/3rkmhzI/8JFCcaZet8+lA2LtOf95jGmwcRwPlNFX0Jdd1QBk2nUAUpCePWIdg7GRxYuO7a4QCbhc6bSyB/WrCQZ1oEmtwGSRqxpMKvQmjxJObg40MgP0DaquGov30A4QLoEdvXCnM7w3QyIr5L/BrLlnCy8LesAZIBn2l5LAHrBFGh9t/lCK9APlUjD9dgT6eCYOSbfdBghKCGsajAsM/fwsPvcjOmpFCLf+UZTer36B57ccHrYdU="
```

Comment:
FYI. Seems we are not the only people doing [this]( https://github.com/grosser/travis_dedup/blame/041373bbe8e07cfa76b5ce96bc21e0d9fe077471/Readme.md#L37 ).